bom/runtime/pom.xml

```diff
@@ -164,6 +164,9 @@
         <antlr.version>4.7.2</antlr.version>
         <quarkus-security.version>1.0.0.Alpha2</quarkus-security.version>
         <javax.interceptor-api.version>1.2</javax.interceptor-api.version>
+        <findbugs.version>3.0.2</findbugs.version>
+        <jsoup.version>1.11.3</jsoup.version>
+        <wagon-provider-api.version>3.0.0</wagon-provider-api.version>
```

Comment:
Are these the versions we want to enforce in user apps?

Reply:
There is not really anything we can do about this now without completely re-structuring our build to have a proper deployment and runtime split in the boms. This is no different to what we are doing elsewhere so I think it is fine for now.

Reply:
How about moving them directly to the dependency management section of build-parent/pom.xml? If that works, that's where they should be for now.